CONTRIBUTING.md

```diff
@@ -2,27 +2,21 @@
 Thanks for your interest in contributing to Supermarket!
 
 The basic process:
-- Sign a Chef CLA (see below)
 - Create a git topic branch for your patch and push it to GitHub
+- Sign your commits with DCO (Developer Certificate of Origin)
 - Open a pull request
 
 ## The Apache License and Chef Contributor License Agreements
 Licensing is very important to open source projects, it helps ensure the software continues to be available under the terms that the author desired. Chef uses the Apache 2.0 license to strike a balance between open contribution and allowing you to use the software however you would like to.
 
 The license tells you what rights you have that are provided by the copyright holder. It is important that the contributor fully understands what rights they are licensing and agrees to them. Sometimes the copyright holder isn't the contributor, most often when the contributor is doing work for a company.
 
-To make a good faith effort to ensure these criteria are met, Chef requires an Individual Contributor License Agreement (ICLA) or a Corporate Contributor License Agreement (CCLA) for all contributions. This is without exception due to some matters not being related to copyright and to avoid having to continually check with our lawyers about small patches.
-
-It only takes a few minutes to complete a CLA, and you retain the copyright to your contribution.
-
-You can [become a contributor by signing the ICLA or by contributing on behalf of your company](https://supermarket.chef.io/become-a-contributor).
-
-For more information about licensing, copyright, and CLAs see Chef's [Community Contributions](https://docs.chef.io/community_contributions.html) page.
+For more information about licensing, copyright, and how to sign your commits with DCO, see Chef's [Community Contributions](https://docs.chef.io/community_contributions.html) page.
```

Comment:
The community contributions page talks about the requirement for DCO sign-offs, but does not currently provide instructions for how to do so. https://github.com/chef/chef/blob/master/CONTRIBUTING.md#dco-sign-off-methods does at the moment.